### PR TITLE
Remove lxd-profile.yaml from charms, but test with a well known profile

### PR DESCRIPTION
**Pull request by addyess**
Removes the lxd-profile.yaml from the k8s and k8s-worker charms, but uses a single profile during integration tests if the substrate is lxd

### CODE REVIEW

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -158,6 +158,17 @@ def switch(self, name: str, path: Path):
         app["channel"] = None
 
 
+async def cloud_profile(ops_test: OpsTest):
+    """Apply lxd-profile to the model if the juju cloud is lxd."""
+    controller = await ops_test.model.get_controller()
+    cloud = await controller.cloud()
+    if cloud.cloud.type_ == "lxd":
+        lxd = LXDSubstrate(None, None)
```
Comment (mateoflorido) [inline tests/integration/conftest.py:166]: Nice!

**Review by mateoflorido — APPROVED** [review]
LGTM!

File: charms/worker/lxd-profile.yaml
Comment (mateoflorido) [inline charms/worker/lxd-profile.yaml:1]: We need to remove the profile in `charms/worker/k8s/lxd-profile.yaml` as well, right?

Comment (addyess) [reply charms/worker/lxd-profile.yaml:1]: hooray!  it's been MOVED to `/tests/integration/data/k8s.profile`

**Pull request merged**